server/grammar/duration.bnf

```diff
@@ -1,8 +1,7 @@
 duration                = (note-length | seconds | milliseconds)
-                          <ows> barline? <ows> subduration* slur?
+                          (<ows> barline? <ows> subduration)*
```

Comment:
Oh yeah -- I guess the first `ows` would greedily consume all of the whitespace.